core/src/main/java/org/elasticsearch/index/query/SpanContainingQueryBuilder.java

```diff
@@ -92,9 +84,16 @@ protected Query doToQuery(QueryParseContext parseContext) throws IOException {
 
     @Override
     public QueryValidationException validate() {
-        QueryValidationException validationExceptions = validateInnerQuery(big, null);
-        validationExceptions = validateInnerQuery(little, validationExceptions);
-        return validationExceptions;
+        QueryValidationException validationException = null;
+        if (big == null) {
+            validationException = addValidationError("inner clause [big] cannot be null.", validationException);
+        }
+        if (little == null) {
+            validationException = addValidationError("inner clause [little] cannot be null.", validationException);
+        }
+        validationException = validateInnerQuery(big, validationException);
+        validationException = validateInnerQuery(little, validationException);
```

Comment:
right, let's not do that then. it just feels weird that we call validate even if the ineer query is null. nothing happens but maybe we should call it only when it is not null?